ppl/cmd/zar/README.md

```diff
@@ -32,26 +32,25 @@ ln -s zq-sample-data/zng
 
 ## ingesting the data
 
-Let's take those logs and ingest them into a directory.   We'll make it
-easier to run all the commands by setting an environment variable pointing
-to the root of the logs tree.
+Let's take those logs and ingest them into a directory.  Often we'd keep our
+archive somewhere like [Amazon S3](https://aws.amazon.com/s3/), but since we're
+just doing a quick test we'll use local temp space.  We'll make it easier to
+run all the commands by setting an environment variable pointing to the root of
+the logs tree.
 ```
-mkdir ./logs
-set ZAR_ROOT=`pwd`/logs
+export ZAR_ROOT=/tmp/logs
+mkdir $ZAR_ROOT
 ```
 
 Now, let's ingest the data using "zar import".  We are working on more
-sophisticated ways to ingest data (e.g., by the standard time partitioning
-techniques of year/month/day/hour etc), but for now zar import just chops
-its input into chunk files of approximately equal size.
-
-Zar import expects its input to be
-in the zng format so we'll use zq to take all the zng logs, gunzip them,
-and feed them to "zar import", which here expects its data on stdin.  We'll chop them
-into chunks of 25MB, which is very small, but in this example the data set is
-fairly small (175MB) and you can always try it out on larger data sets:
+sophisticated ways to ingest data (e.g., arbitrary partition keys and
+auto-sizing of partitions) but for now zar import just chops its input into
+chunk files of approximately equal size, each sorted by timestamp in descending
+order.  We'll chop into chunks of 25MB, which is very small, but in this
+example the data set is fairly small (175MB) and you can always try it out on
+larger data sets:
```

Comment:
@mccanne: I just took a shot at re-wording the `zar import` description to incorporate your suggestion and also reflect current reality as I know it. Could you give it a read-read? Thanks.

Reply:
(@mccanne gave a :+1:  of this reword via Slack. Merging...)